syfertext/vocab.py

```diff
@@ -17,18 +18,19 @@ def __init__(self, model_name: str):
 
         self.model_path = os.path.join(dirname, "SyferText", model_name)
 
-        # Create the 'strings' list that holds all of the words that the Vocab object knows and
+        # Create the 'strings' list that holds words that the Vocab object knows and
         # have vectors for
-        self.strings = self.load_strings()
+        strings = self.load_strings()
+
+        # StringStore object acts as a lookup table for known strings
+        # Stores strings encoded as hash values to save memory
```

Comment:
This comment is not very clear:

```
# Create a `StringStore` object which acts like a lookup table
# mapping between all strings known to the vocabulary and
# their hashes. It can be used to retrieve a string given its hash
# key, or vice versa.
```